context/app/static/js/components/Detail/visualization/Visualization/Visualization.jsx

```diff
@@ -75,73 +93,90 @@ function Visualization(props) {
+                horizontal: 'center',
+              }}
+              open={vizEscSnackbarIsOpen}
+              autoHideDuration={4000}
+              onClose={() => setVizEscSnackbarIsOpen(false)}
+              message="Press [esc] to exit full window."
             />
-          ) : null}
-        </StyledHeaderRight>
-      </StyledHeader>
-      <Paper>
-        <ExpandableDiv $isExpanded={vizIsFullscreen} $theme={vizTheme}>
-          <EscSnackbar
-            anchorOrigin={{
-              vertical: 'top',
-              horizontal: 'center',
-            }}
-            open={vizEscSnackbarIsOpen}
-            autoHideDuration={4000}
-            onClose={() => setVizEscSnackbarIsOpen(false)}
-            message="Press [esc] to exit full window."
-          />
-          {vitessceErrors.length > 0 && (
-            <ErrorSnackbar
+            <VitessceInfoSnackbar
               anchorOrigin={{
-                vertical: 'bottom',
-                horizontal: 'right',
+                vertical: 'top',
+                horizontal: 'center',
               }}
-              open
-              key={vitessceErrors[0]}
-            >
-              <Alert severity="error" onClose={() => removeError(vitessceErrors[0])}>
-                {vitessceErrors[0]}
-              </Alert>
-            </ErrorSnackbar>
-          )}
-          <Vitessce
-            config={vitData[vitessceSelection] || vitData}
-            theme={vizTheme}
-            height={vizIsFullscreen ? null : vitessceFixedHeight}
-            onWarn={addError}
-          />
-        </ExpandableDiv>
-      </Paper>
-      <StyledFooterText variant="body2">
-        Powered by&nbsp;
-        <Link href="http://vitessce.io" target="_blank" rel="noreferrer">
-          Vitessce
-        </Link>
-      </StyledFooterText>
-      <style type="text/css">{vizIsFullscreen ? bodyExpandedCSS : null}</style>
-    </StyledSectionContainer>
+              open={onCopyUrlSnackbarOpen}
+              autoHideDuration={4000}
+              $isWarning={onCopyUrlWarning}
+              onClose={() => setOnCopyUrlSnackbarOpen(false)}
+              message={`Shareable URL copied to clipboard. ${onCopyUrlWarning}`}
+            />
+            {vitessceErrors.length > 0 && (
+              <ErrorSnackbar
+                anchorOrigin={{
+                  vertical: 'bottom',
+                  horizontal: 'right',
+                }}
+                open
+                key={vitessceErrors[0]}
+              >
+                <Alert severity="error" onClose={() => removeError(vitessceErrors[0])}>
+                  {vitessceErrors[0]}
+                </Alert>
+              </ErrorSnackbar>
+            )}
+            <Vitessce
+              config={vitessceConfig[vitessceSelection] || vitessceConfig}
+              theme={vizTheme}
+              onConfigChange={handleVitessceConfigDebounced}
+              height={vizIsFullscreen ? null : vitessceFixedHeight}
```

Comment:
I think there is probably some sort of internal check for truthiness here so I am going to leave this.  I am in general a fan of ternaries because they are clear about what they do.  Especially in jsx, you can end up rendering things unintentionally with && or || if you are not careful.  This change would have broken full-screen mode.